k8s/dragonfly/chart/dragonfly/templates/dfget-daemonset.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Release.Name }}-dfget
+  labels: &DragonflyDfgetLabels
+    app.kubernetes.io/name: {{ .Release.Name }}
+    app.kubernetes.io/component: dragonfly-dfget
+spec:
+  selector:
+    matchLabels: *DragonflyDfgetLabels
+  template:
+    metadata:
+      labels: *DragonflyDfgetLabels
+    spec:
+      hostPID: true  
+      initContainers:
+        - name: wait-for-scheduler
+          image: busybox:latest
+          imagePullPolicy: IfNotPresent
+          command: ['sh', '-c', 'until nslookup {{ .Release.Name }}-scheduler-service && nc -vz {{ .Release.Name }}-scheduler-service {{ .Values.scheduler.grpcPort }}; do echo waiting for scheduler; sleep 2; done;']
+        - name: update-containerd
+          image: dragonflyoss/openssl
+          imagePullPolicy: IfNotPresent
+          #resources:
+
+          command:
+          - /bin/sh
+          - -cx
+          - |-
+            etcContainerd=/host/etc/containerd 
+            if [[ -e $etcContainerd/config.toml ]]; then
+              echo containerd config found
+            else
+              echo $etcContainerd/config.toml not found
+              exit 1
+            fi
+            cat $etcContainerd/config.toml
```

Comment:
do we need to print the config here? by security reasons, not sure if we should send configs to stdout